pysb/tests/test_importers.py

```diff
@@ -9,9 +9,20 @@
 import mock
 import tempfile
 import shutil
+from pysb.logging import get_logger
+
+
+REDUCED_PRECISION = {
```

Comment:
Oh, I just meant to delete the part about the relative sizes of x and y impacting the precision of `x*y**-1`. The rest of that comment is important to keep to help understand why these models need reduced precision in the results comparison.